CHANGELOG.md

```diff
@@ -232,6 +307,30 @@ regression test suite and patches at https://github.com/seqan/seqan3/tree/master
     * `seqan3::get<seqan3::field::cigar>(record)` => `record.cigar_sequence()`
     * `seqan3::get<seqan3::field::tags>(record)` => `record.tags()`
 
+Header Changes:
+
+```cpp
+#include <seqan3/io/{alignment_file=> sam_file}/format_bam.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/format_sam.hpp>
+#include <seqan3/io/{alignment_file => sam_file/detail/}format_sam_base.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/header.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input_format_concept.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/input_options.hpp>
+#include <seqan3/io/{alignment_file/misc => sam_file/sam_flag}.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output_format_concept.hpp>
+#include <seqan3/io/{alignment_file=> sam_file}/output_options.hpp>
+#include <seqan3/io/sam_file/record.hpp> [Added]
+#include <seqan3/io/{alignment_file=> sam_file}/sam_tag_dictionary.hpp>
```

Comment:
```suggestion
#include <seqan3/io/{alignment_file=> sam_file}/format_bam.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/format_sam.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/header.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/input.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/input_format_concept.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/input_options.hpp>
#include <seqan3/io/{alignment_file/misc => sam_file/sam_flag}.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/output.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/output_format_concept.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/output_options.hpp>
#include <seqan3/io/{alignment_file=> sam_file}/sam_tag_dictionary.hpp>
```


#include <seqan3/io/sam_file/record.hpp> [Added] I think we shouldn't include added


#include <seqan3/io/{alignment_file => sam_file/detail/}format_sam_base.hpp>
detail